en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md

```diff
@@ -266,7 +266,7 @@ Follow the instructions below to set up and configure the databases for the WSO2
     ```
    
     !!! Note
-        This configuration is required to disable group and role separation in the WSO2 Identity Server since this is enabled by default.
+        This configuration is required to disable group and role separation in the WSO2 Identity Server since this is enabled by default. If the `GroupAndRoleSeparationEnabled` setting is set to true in the Identity Server, it is necessary to also set `GroupAndRoleSeparationEnabled` to true in the API Manager. 
```

Comment:
Seems we have already added the previous note considering a similar concern in [1]. Since our intension is to ask the user to disable this config in IS side, let's leave the previous explanation as it is. 

[1] https://github.com/wso2/docs-apim/issues/7792